scipy/integrate/_ivp/rk.py

```diff
@@ -3,7 +3,7 @@
 from .base import OdeSolver, DenseOutput
 from .common import (validate_max_step, validate_tol, select_initial_step,
                      norm, warn_extraneous, validate_first_step)
-
+from . import dop853_coefficients
```

Comment:
Just a question from a design standpoint--it looks like `RK45` & `RK23` are also in this module but don't use these coefficients? Since the import executes some top-level code in `dop853_coefficients` to intialize a bunch of values, is there something to be said for confining these operations to only happen within the `DOP853` class initialization?

Reply:
It seems quite hard to control such things. All these classes are imported in `ivp.py`, so I guess these coefficients are going into memory anyway. Doing some "lazy" initialization --- I think it creates unnecessary complexity and does more harm than good. After all the size of the coefficients arrays is negligible.

Another way to organize the code is to put `DOP853` and all the coefficients within one file. But there are some common constants for Runge-Kutta methods, so it's also not ideal.

Reply:
agreed, this is very low-cost so should be fine to keep as is